dags/openshift_nightlies/tasks/install/openshift.py

```diff
@@ -44,6 +44,7 @@ def __init__(self, dag, config: DagConfig, release: OpenshiftRelease):
 
         # Merge all variables, prioritizing Airflow Secrets over git based vars
         self.config = {
+            **self.release.get_latest_release(),
```

Comment:
keith won't this affect the ci though , the nightly versions will be overwritten by "" since they are "" in the git configs.